bezier-rs/lib/src/lib.rs

```diff
@@ -2,8 +2,18 @@ use glam::DVec2;
 
 /// Representation of the handle point(s) in a bezier segment
 pub enum BezierHandles {
-	Quadratic { handle: DVec2 },
-	Cubic { handle1: DVec2, handle2: DVec2 },
+	/// Handles for a quadratic segment
+	Quadratic {
+		/// Point representing the location of the single handle
+		handle: DVec2,
+	},
+	/// Handles for a cubic segment
+	Cubic {
+		/// Point representing the location of the handle associated to the start point
+		handle1: DVec2,
```

Comment:
Thought: For further clarification you could name these `handle_start ` and `handle_end` something similar based on your lexicon. The comments do help clarify, but could be easily lost when interacting with the system as an opaque library.